projects/meta_cl/experiments/oml.py

```diff
@@ -79,8 +79,91 @@ def run_experiment(config):
     run_meta_test=True,
 )
 
+# This attempts to replicate the results from the original OML paper.
+# See results below.
+metacl_oml_replicate = copy.deepcopy(metacl_oml)
+metacl_oml_replicate.update(
+
+    # How often to checkpoint (in epochs)
+    checkpoint_freq=1000,
+    keep_checkpoints_num=1,
+    checkpoint_at_end=True,
+    checkpoint_score_attr="training_iteration",
+
+    # Number of tasks (i.e. distinct classes) for the inner loops and outer loop
+    tasks_per_epoch=3,
```

Comment:
Is it just 3?

Reply:
It is, this is taken from the OML readme where they suggest running the following to replicate their results:

```
python oml_omniglot.py --update_lr 0.03 --meta_lr 1e-4 --name OML_Omniglot/ --tasks 3 --update_step 5 --steps 700000 --rank 0
```

Notice `--tasks 3`. The results reported in `projects/meta_cl/experiments/oml.py` come from running OML's code with this configuration. So the comparison is fair in any case.

Reply:
I see. I don't remember seeing the 3 anywhere in the paper, but I think they just don't mention that part. For some reason I had 5 in my mind.